homework/homework7/bulls_and_cows.py

```diff
@@ -0,0 +1,60 @@
+"""Module game 'Bulls and Cows'.
+The program guesses a sequence of numbers.
+"""
+import random
+
+# Количество цифр в загадываемом слове
+COUNT = 4
+
+
+# pylint: disable=invalid-name
+def generate_random_number(count):
+    """This function generates a random number with non-repeating digits."""
+    # Создаем список цифр от 0 до 9
```

Comment:
You do not need to write comments for each line. Use doc-strings and write comment only when it's really needed

Reply:
Ok, thanks)